src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -365,8 +365,7 @@ def _ignore_if_not_registered(self, dest):
         if not match:
             super().argument(dest, arg_type=ignore_type)
 
-    # pylint: disable=arguments-differ
```

Comment:
Putting `# pylint: disable=arguments-differ` here is incorrect as it disables all `arguments-differ` below, not only for `argument()`.